app/controllers/admin/products_controller.rb

```diff
@@ -1,6 +1,15 @@
 module Admin
   class ProductsController < AdminController
 
+    def index
+      @source_models = Product.where(state: 'active')
```

Comment:
since you have https://github.com/wantable/ar_state_machine/ for products you can just do 
```@source_models = Product.active``` or even ```@source_models = Product.where(state: Product::ACTIVE)```